inc/saitransceiver.h

```diff
@@ -835,147 +835,145 @@ typedef enum _sai_transceiver_stat_t
      * @unit dBm
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_OUTPUT_POWER,
 
     /**
      * @brief Laser bias current
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_LASER_BIAS_CURRENT,
 
     /**
      * @brief Transceiver temperature
      *
      * @type sai_uint64_t
-     * @precision precision2
-     * @precision precision1
      * @iscounter false
+     * @precision precision1
      */
     SAI_TRANSCEIVER_STAT_TEMPERATURE,
 
     /**
      * @brief Bit error rate before forward error correction
      *
      * @type sai_uint64_t
-     * @precision precision18
      * @iscounter false
+     * @precision precision18
      */
     SAI_TRANSCEIVER_STAT_PRE_FEC_BER,
 
     /**
      * @brief Bit error rate after forward error correction
      *
      * @type sai_uint64_t
-     * @precision precision18
      * @iscounter false
+     * @precision precision18
      */
     SAI_TRANSCEIVER_STAT_POST_FEC_BER,
 
     /**
      * @brief EDFA bias current
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_EDFA_BIAS_CURRENT,
 
     /**
      * @brief Laser bias voltage
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_LASER_BIAS_VOLTAGE,
 
     /**
      * @brief Case temperature
      *
      * @type sai_uint64_t
-     * @precision precision2
-     * @precision precision1
      * @iscounter false
+     * @precision precision1
      */
     SAI_TRANSCEIVER_STAT_CASE_TEMPERATURE,
 
     /**
      * @brief Power consumption
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_POWER_CONSUMPTION,
 
     /**
      * @brief Tx mod bias XI
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_TX_MOD_BIAS_XI,
 
     /**
      * @brief Tx mod bias XQ
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_TX_MOD_BIAS_XQ,
 
     /**
      * @brief Tx mod bias YI
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_TX_MOD_BIAS_YI,
 
     /**
      * @brief Tx mod bias YQ
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
      */
     SAI_TRANSCEIVER_STAT_TX_MOD_BIAS_YQ,
 
     /**
      * @brief Tx mod bias XPH
      *
      * @type sai_uint64_t
-     * @precision precision2
      * @iscounter false
+     * @precision precision2
```

Comment:
duplicating precision2 with @precision tag is not necessary, maybe alone "2" will be sufficient ?